libvast/src/system/indexer_downstream_manager.cpp

```diff
@@ -72,10 +93,19 @@ int32_t indexer_downstream_manager::max_capacity() const noexcept {
 void indexer_downstream_manager::register_partition(partition* p) {
   VAST_DEBUG(self(), "registers partition", p->id());
   partitions.insert(p);
+  // Corner case: it is possible that register gets called
```

Comment:
I don't quite understand this comment.

Reply:
I somehow forgot to finish that, updated.